vowels-component.jsx

```diff
@@ -0,0 +1,100 @@
+import * as React from "react";
+import "./styles.css";
+
+export default function App() {
+  const [name, setName] = React.useState("");
+  const [records, setRecords] = React.useState([]);
+  const vowelsfreq = (name) => {
+    const count = name.match(/[aeiou]/gi)?.length || 0;
+    return count;
+  };
+  const vFreq = vowelsfreq(name);
+  const newRecords = [...records];
```

Comment:
This `newRecords` is used inside the function `handleSubmit` and not anywhere else in this component.
So please declare it inside that function only. Same for `vFreq` variable.

Less variables in the component outside the functions, better the code review experience will be.